lib/net.c

```diff
@@ -1333,17 +1331,29 @@ gatekeeper_init_network(struct net_config *net_conf)
 	if (net_conf == NULL)
 		return -1;
 
+	gatekeeper_logtype = rte_log_register("gatekeeper");
+	if (gatekeeper_logtype < 0) {
```

Comment:
Registering the `gatekeeper` log type here is too late. `config_gatekeeper()`, for example, may log something before reaching this line. My suggestion is to register the `gatekeeper` log type just after Gatekeeper gets the options from the command line. Thanks to the latest merged pull request, it's now easy to add options to the command line. Please add an option for the log level that the `gatekeeper` log type should be initialized, and another option for the global log level.

Reply:
You are right, reusing EAL here is better. Please document those parameters on the wiki because that is not an obvious solution.